apps/communications/contacts/locales/contacts.en-US.properties

```diff
@@ -110,9 +112,9 @@ exportContactsButton    = Export Contacts
 exportContactsAction    = Export
 
 # Export
-simExport-title        = Export to SIM
-memoryCardExport-title = Export to memory card
-btExport-title         = Export to Bluetooth
+simExport-title        = Exporting
+memoryCardExport-title = Exporting
+btExport-title         = Exporting
```

Comment:
You can't changes strings without also changing the l10nId key. It messes up our localizers if you do that.

Also, it seems premature to me to be making this kind of string changes.  The UX we have from Litehouse so far seems like wireframes. I would expect final text to come along with final visual specs...  Let's not change too many strings now unless you've double-checked that these are what is really wanted.  I assume that some of the stuff in the wireframes is just placeholder text.